gadgets/dicom/CMakeLists.txt

```diff
@@ -83,6 +83,10 @@ target_link_libraries(gadgetron_dicom
     ISMRMRD::ISMRMRD
     ${GT_DICOM_LIBRARIES} )
 
+if (WIN32)
+    target_link_libraries(gadgetron_dicom netapi32.lib wsock32.lib WS2_32.Lib)
```

Comment:
This should not be required at all, and will almost certainly break in the future.
Have you tried just using dcmtk from VCPKG? It should link to the dependencies automatically.